docs/examples.md

```diff
@@ -45,6 +45,21 @@ data-validation validate column -sc my_bq_conn -tc my_bq_conn -tbls bigquery-pub
 ````shell script
 data-validation validate row -sc my_bq_conn -tc my_bq_conn -tbls bigquery-public-data.new_york_citibike.citibike_stations --primary-keys station_id --hash '*'
 ````
+#### Run a row hash validation for all rows when database is Oracle and primary key is of Numeric Datatype
+Typecast Numeric to String by either of following: 
```

Comment:
Let's add some details here i.e "For Oracle and Postgres connections, numeric primary keys may not match between source and target data types due to precision or scale discrepancies. You may need to cast the primary key to a string for a valid match."